src/test/java/io/lettuce/core/commands/ScriptingCommandIntegrationTests.java

```diff
@@ -134,7 +134,7 @@ void evalWithArgs() {
     @EnabledOnCommand("EVAL_RO") // Redis 7.0
     void evalReadOnly() {
         String[] keys = new String[] { "key1" };
-        assertThat((String) redis.evalReadOnly("return KEYS[1]".getBytes(), STATUS, keys, "a")).isEqualTo("key1");
+        assertThat((String) redis.evalReadOnly("return KEYS[1]", STATUS, keys, "a")).isEqualTo("key1");
```

Comment:
We are reusing the original test to let the implementation properly encode the script.

Reply:
Sure, makes sense